sources/Launcher/DYNMarginCalculationLauncher.h

```diff
@@ -218,6 +219,13 @@ class MarginCalculationLauncher : public RobustnessAnalysisLauncher {
    */
   void initLog();
 
+  /**
+   * @brief Compute the scenario IIDM file corresponding to a variation
```

Comment:
it generates the name of the corresponding iidm file

Reply:
OK